eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -197,6 +199,10 @@ function Upload-Blobs
     Write-Host "DocDir $($DocDir)"
     Write-Host "Final Dest $($DocDest)/$($PkgName)/$($DocVersion)"
 
+    # Use the step to replace master link to release tag link 
+    Write-Host "Replacing all readme master links with release tag $Tag"
+    ReplaceLink -scanFolder $DocDir -fileSuffix ".html" -replacement $Tag -customRegex $RepoReplaceRegex
```

Comment:
Don't you need to source in the script to use this function?

Reply:
Added back. A miss when copy the function from test branch back.

Reply:
This should be as simple as:
```
foreach ($htmlFile in (Get-ChildItem $DocDir -include *.html -r)) 
{
  $fileContent = Get-Content $htmlFile
  $updatedFileContent = $fileContent -replace $repoReplacRegex, "`${1}$Tag`$2"
  if ($fileContent -ne $updatedFileContent) {
    $updatedFileContent = Set-Content $htmlFile
  }
}
```
Maybe we can refactor that into a shared function if we can share it otherwise it seems simple enough to just have it here.

Reply:
@sima-zhu  I think you might have missed this comment.